pkg/linux/build-functions.sh

```diff
@@ -18,6 +18,19 @@ _setup_env() {
     if [ -n "${APP_SUFFIX}" ]; then
         APP_LONG_VERSION=${APP_LONG_VERSION}-${APP_SUFFIX}
     fi
+
+    # Setting up the correct Python version for Ubuntu 18 and EL-8, which have
+    # new Python versions installed parallel to the old ones.
+    OS_VERSION=$(grep "^VERSION_ID=" /etc/os-release | awk -F "=" '{ print $2 }' | sed 's/"//g' | awk -F "." '{ print $1 }')
+    SYSTEM_PYTHON_PATH='/usr/bin/python3'
+    PYTHON_BINARY='python3'
+    if [ "${OS_VERSION}" == 18 ]; then
```

Comment:
As this is in the shared functions file, we can't assume it's being run on Ubuntu or EL. The test should look more like:

if [ "$1" == "redhat" ] && [ "${OS_VERSION}" == "8" ]; then

Reply:
Additionally, you're comparing a string to an int. Quote the 18 :-)